### ISSUE TITLE
No updates from server

### ISSUE THREAD

**Issue by NKarolak** [corona-warn-app/cwa-app-android#4723]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: Samsung S10e
- Android version: 12
- App version: 2.16.2
- ENF 18214815000

## Describe the bug

Since yesterday about 6 PM CET, the CWA does not receive any more contact keys from the server. 
There's no error, just no new data after a short check
![Screenshot_20220123-092800_Corona-Warn](https://user-images.githubusercontent.com/34504100/150670590-78420344-59b6-42a5-aed0-4d6c6724b50b.jpg)
My boyfriend's Huawei phone is also affected.

## Steps to reproduce the issue

Open the CWA and check "Updated" in the green or read notification box.

**Comment by thomasaugsten** [comment]
Please be aware on mobile data the app is refreshing only once per day and please double check your wifi connection

**Comment by NKarolak** [comment]
> Please be aware on mobile data the app is refreshing only once per day and please double check your wifi connection

Both phones were are on WI-FI non stop since last night.

**Comment by thomasaugsten** [comment]
You can activate error report under app information and wait 6h. You can have a look by your own into the error report or submit the report and provide the error log id.

**Comment by NKarolak** [comment]
Okay, report is running now. See you later 🙂

**Comment by swekkx** [comment]
Same issue for me. Since yesterday 3pm no Synchronisation on my Android S21. Smartphone is connected to the WiFi 👍

**Comment by NKarolak** [comment]
@thomasaugsten What are the log keywords to look for?

**Comment by thomasaugsten** [comment]
Error or timeout but we investigate this in parallel

**Comment by MikeMcC399** [comment]
I am seeing the same issue. There is no record of a recent exposure check in the Google exposure check logs, even though I saw CWA doing an exposure check. CWA shows "Updated: Yesterday, 18:05". (CWA 2.16.2, Android 11, ENS: v1.8 (214815000).

On a second device which I keep for testing, I reset the app and it successfully carried out an exposure check, recorded this in the UI of CWA and in the Google exposure check log. That means I have connectivity (through WiFi) to the server.

**Comment by NKarolak** [comment]
@thomasaugsten What about that?

```
2022-01-23T08:56:24.252Z V/DefaultExposureDetectionTracker:HD: Waiting for update.
2022-01-23T08:56:24.251Z V/DefaultExposureDetectionTracker:HD: init()
2022-01-23T08:56:24.252Z V/DefaultExposureDetectionTracker$detectionStates$2$setupAutoSave: Observing detection changes.
2022-01-23T08:56:24.252Z V/DefaultExposureDetectionTracker:HD: Internal onStart
2022-01-23T08:56:24.252Z V/DefaultExposureDetectionTracker:HD: Providing startValue...
2022-01-23T08:56:24.255Z V/ExposureDetectionTrackerStorage: Json read from /data/user/0/de.rki.coronawarnapp/files/calcuation_tracker/calculations.json
2022-01-23T08:56:24.255Z V/ExposureDetectionTrackerStorage: Loaded detection data: {5a2c8fcd-d968-478e-9447-dc0dad1c18ce=TrackedExposureDetection(identifier=5a2c8fcd-d968-478e-9447-dc0dad1c18ce, startedAt=2022-01-21T15:25:36.156Z, result=NO_MATCHES, finishedAt=2022-01-21T15:26:12.493Z, enfVersion=V2_WINDOW_MODE), 3dba434e-7a27-4f58-b215-612c81bb6e5d=TrackedExposureDetection(identifier=3dba434e-7a27-4f58-b215-612c81bb6e5d, startedAt=2022-01-21T19:32:24.911Z, result=NO_MATCHES, finishedAt=2022-01-21T19:32:53.374Z, enfVersion=V2_WINDOW_MODE), dd446248-bf2e-4016-b1ba-9fbe6d67184b=TrackedExposureDetection(identifier=dd446248-bf2e-4016-b1ba-9fbe6d67184b, startedAt=2022-01-22T06:19:50.094Z, result=NO_MATCHES, finishedAt=2022-01-22T06:20:20.266Z, enfVersion=V2_WINDOW_MODE), f7c8d7d5-8c42-405b-866f-a8568854432e=TrackedExposureDetection(identifier=f7c8d7d5-8c42-405b-866f-a8568854432e, startedAt=2022-01-22T11:16:44.824Z, result=NO_MATCHES, finishedAt=2022-01-22T11:17:22.314Z, enfVersion=V2_WINDOW_MODE), f6feb054-a75d-4917-8b27-3e8ab5747cb2=TrackedExposureDetection(identifier=f6feb054-a75d-4917-8b27-3e8ab5747cb2, startedAt=2022-01-22T15:20:09.735Z, result=NO_MATCHES, finishedAt=2022-01-22T15:20:37.376Z, enfVersion=V2_WINDOW_MODE)}
2022-01-23T08:56:24.256Z V/DefaultExposureDetectionTracker:HD: ...startValue provided, emitting...
2022-01-23T08:56:24.256Z V/DefaultExposureDetectionTracker:HD: ...startValue provided and emitted.
2022-01-23T08:56:24.256Z V/ExposureDetectionTrackerStorage: Data didn't change, skipping save.
2022-01-23T08:56:24.257Z D/DownloadDiagnosisKeysTask: Next detection is available in -816 min
2022-01-23T08:56:24.257Z D/DownloadDiagnosisKeysTask: isUpdateToEnfV2: false
2022-01-23T08:56:24.257Z V/DefaultExposureDetectionTracker$detectionStates$2$setupTimeoutEnforcer: Running timeout check (now=2022-01-23T08:56:24.256Z): [TrackedExposureDetection(identifier=5a2c8fcd-d968-478e-9447-dc0dad1c18ce, startedAt=2022-01-21T15:25:36.156Z, result=NO_MATCHES, finishedAt=2022-01-21T15:26:12.493Z, enfVersion=V2_WINDOW_MODE), TrackedExposureDetection(identifier=3dba434e-7a27-4f58-b215-612c81bb6e5d, startedAt=2022-01-21T19:32:24.911Z, result=NO_MATCHES, finishedAt=2022-01-21T19:32:53.374Z, enfVersion=V2_WINDOW_MODE), TrackedExposureDetection(identifier=dd446248-bf2e-4016-b1ba-9fbe6d67184b, startedAt=2022-01-22T06:19:50.094Z, result=NO_MATCHES, finishedAt=2022-01-22T06:20:20.266Z, enfVersion=V2_WINDOW_MODE), TrackedExposureDetection(identifier=f7c8d7d5-8c42-405b-866f-a8568854432e, startedAt=2022-01-22T11:16:44.824Z, result=NO_MATCHES, finishedAt=2022-01-22T11:17:22.314Z, enfVersion=V2_WINDOW_MODE), TrackedExposureDetection(identifier=f6feb054-a75d-4917-8b27-3e8ab5747cb2, startedAt=2022-01-22T15:20:09.735Z, result=NO_MATCHES, finishedAt=2022-01-22T15:20:37.376Z, enfVersion=V2_WINDOW_MODE)]
2022-01-23T08:56:24.257Z W/DownloadDiagnosisKeysTask: Aborting. Last detection is recent (<24h) and no new keyfiles.
2022-01-23T08:56:24.257Z I/DownloadDiagnosisKeysTask: task aborted, last check was within 24h, and there are no new files
2022-01-23T08:56:24.257Z I/DownloadDiagnosisKeysTask: Finished (isCanceled=false).
```

**Comment by swekkx** [comment]
Seems to be fixed? I was able to synchronize just few minutes ago.

**Comment by NKarolak** [comment]
@swekkx Right :-)

**Comment by Andiadm** [comment]
Here the problem persists, last update yesterday 16:27 CET.
Also the statistics (Incidece, R-Value etc.) show the values from 2 days ago and are not updated.

**Comment by thomasaugsten** [comment]
Yes it should now be fixed. There was a short interruption in publish the keys.

**Comment by MikeMcC399** [comment]
@NKarolak 
Well done for finding the right place in the log file illustrating the problem! 👍🏻

**Comment by NKarolak** [comment]
@MikeMcC399 Thanks, but was it really the relevant place ...?

**Comment by thomasaugsten** [comment]
It is correct. It states no update because no new keyfile after successful download are provided

**Comment by NKarolak** [comment]
Lucky me :-)
Thanks for your quick intervention.

**Comment by Ein-Tim** [comment]
@thomasaugsten 

Do you know the consequences of this issue? 

Was only the distribution affected or also users who wanted to upload their tests?

**Comment by thomasaugsten** [comment]
Only distributed. consequence was only a delay of the hourly key packages

**Comment by NKarolak** [comment]
@thomasaugsten Seems like the issue is back since recently. Can you check?

**Comment by Andiadm** [comment]
Here the last download was today, 16:47 CET. 
Maybe the servers cant cope with the huge number of positive tests?

**Comment by NKarolak** [comment]
Okay, NOW it has been updated 😁

**Comment by Ein-Tim** [comment]
@NKarolak I guess you experienced https://github.com/corona-warn-app/cwa-app-android/issues/4472 :D

**Comment by NKarolak** [comment]
@Ein-Tim 
Probably. On reopening the app, it looked like it was (shortly) searching for new keys, but didn't find any. Exactly like last time.
However, from now on I will get nervous after **full** 5 hours only, I promise 😁

**Comment by Ein-Tim** [comment]
No worries, better report too early then too late!

**Comment by swekkx** [comment]
Hi again, I am facing the same issue again.

Error report
ID 8A16FD407C2934A7D7B3 

No update since 9 am today. Already restarted my phone.

**Comment by MikeMcC399** [comment]
@swekkx 
> No update since 9 am today. Already restarted my phone.

Can you confirm that you are connected to WiFi all the time? Which Android version is your S21 running?

**Comment by swekkx** [comment]
Yes I am connected to wifi and tried to update the whole day. My phone has android 12 with the most current patches.

It searches for 1 sec and then just show the status view.

![Screenshot_20220202-200457_Corona-Warn](https://user-images.githubusercontent.com/96697098/152220708-d2d16e8c-585e-4fdc-97a2-2e7a0277db1a.jpg)

**Comment by Ein-Tim** [comment]
@swekkx 

Please add your comments to https://github.com/corona-warn-app/cwa-app-android/issues/4472 and continue there.

**Comment by MikeMcC399** [comment]
@NKarolak 

Could we close this issue, since it was a one-off server problem which is fixed?

It was a server outage on Jan 22 and Jan 23, 2022. I can see from my own Google exposure check log that the last check was 18:07 on Saturday, Jan 22, 2022 and the next check was at 11:01 on Sunday, Jan 23, 2022.

As @Ein-Tim noted, there is another issue #4472 for on-going problems.

**Comment by NKarolak** [comment]
@MikeMcC399 I closed this issue already a week ago 😉

**Comment by MikeMcC399** [comment]
@NKarolak 
> I closed this issue already a week ago

My apologies for not paying attention! 😕

**Comment by NKarolak** [comment]
@MikeMcC399 No checks since yesterday 4 PM for me.
Is it again a server issue?

**Comment by Ein-Tim** [comment]
@NKarolak 

On my iPhone last check is at 06:36. 
I suspect that CWA 2.19 made the situation reported in #4472 even worse.

**Comment by NKarolak** [comment]
@Ein-Tim Thanks, I'll follow the other issue, then.

**Comment by MikeMcC399** [comment]
@NKarolak 

> No checks since yesterday 4 PM for me. Is it again a server issue? CWA 2.19.0

I don't have CWA Android 2.19.0 yet and I am still waiting for the Google Play Store to give me access to it due to the staged rollout.

On my main device which is still using 2.18.1 I see no issue:

- Google Exposure Notification System v1.8 (220618000)
- Android 11

The device is connected to WiFi and has been doing regular exposure checks at 4 hour intervals. The last one was at 06:28 today, March 18, 2022.

- According to #4472 it seems that exposure checks are not working reliably on all devices, however this is not the same issue as this closed one which affected all devices.
